environment/src/main/java/com/sequenceiq/environment/environment/repository/EnvironmentRepository.java

```diff
@@ -59,4 +59,9 @@
     @CheckPermission(action = ResourceAction.READ)
     @Query("SELECT e.resourceCrn FROM Environment e WHERE e.name = :name AND e.accountId = :accountId")
     Optional<String> findResourceCrnByNameAndAccountId(@Param("name") String name, @Param("accountId") String accountId);
+
+    @CheckPermission(action = ResourceAction.READ)
+    @Query("SELECT e FROM Environment e WHERE e.network.id = :networkId")
+    Optional<Environment> findByNetworkId(@Param("networkId") Long networkId);
```

Comment:
is it used somewhere ?

Reply:
Most likely not, seems like a leftover method from the beginning. Going to remove it also, thanks